lib/plsql/connection.rb

```diff
@@ -197,9 +197,8 @@ def session_id
       @session_id ||= select_first("SELECT TO_NUMBER(USERENV('SESSIONID')) FROM dual")[0]
     end
 
-    # Set time zone (default taken from TZ environment variable)
+    # Set time zone
     def set_time_zone(time_zone=nil)
-      time_zone ||= ENV['TZ']
```

Comment:
`local` will reset to the initial value which has been at session start. So if we have `ORA_SDTZ=Europe/London` - then set to `local` will keep DST-awareness.

The question is not for plsql-spec own tests but for ruby-plsql API in general:
 - what behavior do we expect from `connection.set_time_zone`? Do we have default, do we allow passing nil value, what is it supposed to do?
 - There might be existing code relying on the old version (i.e. the TZ environment variable). We need to document our change somewhere and perhaps plan bumping up the gem version number accordingly.